garak/probes/web_injection.py

```diff
@@ -200,15 +205,15 @@ def __init__(self, config_root=_config):
         ]
 
 
-class MdExfil20230929(garak.probes.Probe):
+class PlaygroundMarkdownExfil(garak.probes.Probe):
```

Comment:
a `fixer` class should be added for this, to update configs. the existing ones are fine examples